packages/core/src/modules/credentials/formats/indy/IndyCredentialFormat.ts

```diff
@@ -2,6 +2,7 @@ import type { LinkedAttachment } from '../../../../utils/LinkedAttachment'
 import type { CredentialPreviewAttributeOptions } from '../../models'
 import type { CredentialFormat } from '../CredentialFormat'
 import type { IndyCredProposeOptions } from './models/IndyCredPropose'
+import type * as Indy from 'indy-sdk'
```

Comment:
Can't we just import the specific types here?

Reply:
Yes of course, I often import it as indy to indicate we're using the indy sdk types